napari/benchmarks/benchmark_slicing.py

```diff
@@ -0,0 +1,169 @@
+# See "Writing benchmarks" in the asv docs for more information.
+# https://asv.readthedocs.io/en/latest/writing_benchmarks.html
+# or the napari documentation on benchmarking
+# https://github.com/napari/napari/blob/main/docs/BENCHMARKS.md
+
+import time
+
+import numpy as np
+import zarr
+from qtpy.QtWidgets import QApplication
+
+import napari
+from napari.layers import Image
+
+SAMPLE_PARAMS = {
+    'skin_data': {
```

Comment:
I would have expected the benchmark data shape/chunk/dtype to come from one of the examples used in the scripts, but maybe there will be more consistency later?

Reply:
I took this form the napair-sample-data plugin.

I agree, though, that these should match other examples we use.

Reply:
I think we can update this later if needed.